src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -38,13 +38,13 @@
 
     @Override
     public boolean equals(Object o) {
+        Item item = (Item) o;
```

Comment:
Declarations should be at the beginning of the block which they are used rather than waiting until their first use in the block. This helps reduce confusion for people less familiar with the code. Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3